doctr/utils/metrics.py

```diff
@@ -344,13 +344,13 @@ def summary(self) -> Tuple[Optional[float], Optional[float], Optional[float]]:
         """
 
         # Recall
-        recall = self.matches / self.num_gts if self.num_gts > 0 else None
+        recall = self.matches / self.num_gts if self.num_gts > 0 else 0
 
         # Precision
-        precision = self.matches / self.num_preds if self.num_preds > 0 else None
+        precision = self.matches / self.num_preds if self.num_preds > 0 else 0
 
         # mean IoU
-        mean_iou = self.tot_iou / self.num_preds if self.num_preds > 0 else None
+        mean_iou = self.tot_iou / self.num_preds if self.num_preds > 0 else 0
```

Comment:
I understand that keeping it this way would crash the script, but perhaps logging a `0` isn't a good idea either?
There is a difference between having a metric being undefined and being 0 (I'm afraid we might miss something later on)